src/index.tsx

```diff
@@ -5,7 +5,7 @@ import ReactDOM from 'react-dom/client';
 import { makeServer } from './server';
 import Theme from './theme';
 
-import Routes from './routes';
+import AppRoutes from './routes';
```

Comment:
**question:** Do we have any reason now to be renaming this?

Reply:
Change it to not have a problem with the Routes mapper